osx/install_swift_binaries.sh

```diff
@@ -36,7 +36,7 @@ brew install wget > /dev/null || brew outdated wget > /dev/null || brew upgrade
 
 #Download and install Swift
 echo "Swift installed $SWIFT_PREINSTALL does not match snapshot $SWIFT_SNAPSHOT."
-wget -nv https://swift.org/builds/$SNAPSHOT_TYPE/xcode/$SWIFT_SNAPSHOT/$SWIFT_SNAPSHOT-osx.pkg
+wget --no-verbose https://swift.org/builds/$SNAPSHOT_TYPE/xcode/$SWIFT_SNAPSHOT/$SWIFT_SNAPSHOT-osx.pkg
```

Comment:
So in the meantime, Ian had added `-nv` for the swift download in #138.  This change is fine but could we keep `linux/install_swift_binaries.sh` consistent too?

Reply:
I'd already done the change but missed it when adding files. I changed both to be --no-verbose as I prefer the more descriptive option that doesn't require any look-ups as to what it may mean.